amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Utility class with helpful functions for the Amaranth Chrome extension.
+ * All methods are static.
+ */
+class AmaranthUtil {
+  /**
+   * Removes special characters from a string.
+   * @param {string} str String to remove special characters from
+   * @param {string} filters Characters to remove from str
+   * @return {string} `str` with all characters in `filters` removed
+   */
+  static removeSpecialCharacters(
+      str, filters='!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~\t\n') {
+    let newStr = '';
+    for (const char of str) {
+      if (!filters.includes(char)) {
+        newStr += char;
+      }
+    }
+
+    return newStr;
+  }
+
+  /**
+   * Pads some array `arr` with `padValue` until it's length is equal to
+   * `desiredLength`. This function will modify the original array `arr` and
+   * return it. If the length of `arr` is already greater than `desiredLength`,
+   * the original `arr` is returned unchanged.
+   * @template T
```

Comment:
You can likely remove this empty doc tag since there's no additional info provided.

Reply:
This is actually [defining a generic (or template) type](https://google.github.io/styleguide/jsguide.html#features-functions-generics), `T`, that's used to specify the types of `arr`, `padValue`, and the return type below. Removing it removes code editor type hints and potentially muddies the docs when generated. I will add some additional info for it to make it a little more useful in plaintext format

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/9477cd22ab09c4810ffb8663d7ff55b8d366f615